setup.cfg

```diff
@@ -92,6 +92,7 @@ universal = 1
 filterwarnings =
     error::DeprecationWarning:mopidy[.*]
     ignore::PendingDeprecationWarning:mopidy[.*]
+    ignore::DeprecationWarning:mopidy[.*]
```

Comment:
This sends mixed signals when compared with two lines up ;-)

Instead of ignoring our own deprecation warnings, we should update the tests:
- If the test is just using the deprecated functionality to support whatever else it is testing, use the replacement for the deprecated feature instead.
- If the test is testing the deprecated functionality, wrap the call in `with deprecation.ignore():` to ignore deprecation warnings in as little code as possible.